shotover-proxy/tests/test-configs/redis/passthrough-valkey/docker-compose.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+  valkey-one:
+    image: chainguard/valkey:latest
```

Comment:
Need to pin a specific version to avoid test breakage

Reply:
Updated description